config/locales/en/application.yml

```diff
@@ -3,21 +3,6 @@ en:
     modal_error:
       title: Something has gone wrong
       description_govspeak: Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
-    phase_banner:
-      raise_a_support_request: Raise a support request
-      send_us_feedback: Send us feedback
-      whats_new: What’s new
-      request_training: Request training
     footer:
       support_and_feedback: Support and feedback
```

Comment:
Can we remove these ones too?

Reply:
(I lament that links didn't make it into ADR #4, as the end result is quite confusing ☹️ .)

Reply:
Those aren't links those are headers: https://github.com/alphagov/content-publisher/blob/e76ed11f777c74936edd2817b717c58c5fc4f5aa/app/views/layouts/application.html.erb#L148

I did umm and ahh about whether to do those as well but seemed out of spirit with the ADR.

> (I lament that links didn't make it into ADR #4, as the end result is quite confusing ☹️ .)

😄 the joys of compromise!

Reply:
Oh yes, sorry.